webpack.prod.config.js

```diff
@@ -108,6 +108,7 @@ plugins: [
       ARC_IPFSPROVIDER_PORT : "",
       ARC_IPFSPROVIDER_PROTOCOL : "",
       ARC_IPFSPROVIDER_API_PATH : "",
+      INFURA_ID : "",
```

Comment:
I think you need to add this to app.json too for review apps to work

Reply:
though review app is working without it ..

Reply:
> I think you need to add this to app.json too for review apps to work

@tibetsprague That is exactly why I thought yesterday that SHOW_ALL_DAOS would not work in @orenyodfat's PR, because he hadn't added SHOW_ALL_DAOS here.  But then I see that he has in fact added it.  So I wonder when he did, before or after he proved that the functionality was working on review apps.

One thing I know:  You have to have these set here for running locally, where heroku doesn't exist to insert values.